modules/verification-tools/blog-verification-tools.php

```diff
@@ -35,126 +35,3 @@ function jetpack_verification_options_init() {
-				} else {
-					$verification_code = $verification_services_codes["$name"];
-				}
-				$ver_tag = sprintf( '<meta name="%s" content="%s" />', esc_attr( $service["key"] ), esc_attr( $verification_code ) );
-				/**
-				 * Filter the meta tag template used for all verification tools.
-				 *
-				 * @module verification-tools
-				 *
-				 * @since 3.0.0
-				 *
-				 * @param string $ver_tag Verification Tool meta tag.
-				 */
-				$ver_output .= apply_filters( 'jetpack_site_verification_output', $ver_tag );
-				$ver_output .= "\n";
-			}
-		}
-	echo $ver_output;
-	}
-}
-add_action( 'wp_head', 'jetpack_verification_print_meta', 1 );
-
-function jetpack_verification_options_form() {
-	$verification_services_codes = get_option( 'verification_services_codes' );
-	?>
-<form method="post" action="options.php">
-	<?php settings_fields( 'verification_services_codes_fields' ); ?>
-	<div class="tools-container">
-	<?php
-	foreach ( jetpack_verification_services() as $key => $service ) {
-		echo "<div class='jp-verification-service'>
-				<h4>" . esc_html( $service['name'] ) . "</h4>
-					<input value='" . esc_attr( isset( $verification_services_codes[ $key ] ) ? $verification_services_codes[ $key ] : '' ) . "' name='verification_services_codes[" . esc_attr( $key ) . "]' type='text' />
-				<small>
-					<label for='verification_services_codes[" . esc_attr( $key ) . "]'>" . esc_html( __( 'Example:' , 'jetpack' ) ) . " <span>&lt;meta name='" . esc_attr( $service['key'] ) . "' content='<strong>" . esc_attr( $service['format'] ) . "</strong>'&gt;</span></label>
-				</small>
-			</div>";
-	}
-	?>
-	</div>
-	<p class="submit">
-		<input type="submit" class="button-primary" value="<?php _e( 'Save Changes' , 'jetpack' ); ?>" />
-	</p>
-</form>
-
-<style>
-/*  Jetpack styles aren't loaded in the tools section of the admin, let's save on some http requests and just do an inline block */
-
-	.jp-verification-tools h3 a {
-		text-decoration: none;
-	}
-
-	.jp-verification-service {
-		border-bottom: 1px #f1f1f1 solid;
-		padding-bottom: 20px;
-	}
-
-	.jp-verification-service input[type="text"] {
-		width: 100%;
-		margin-bottom: 10px;
-	}
-
-	.jp-verification-service label {
-		font-size: 13px;
-	}
-
-	/* mimic 'code' tag style, but this allows for better visuals + line breaks on mobile devices */
-	.jp-verification-service span {
-		display: block;
-		margin-top: 5px;
-		font-size: 14px;
-		padding: 10px;
-		background: #f1f1f1;
-		font-family: monospace;
-		word-wrap: break-word;
-	}
-
-	.jp-verification-service strong {
-		font-weight: bold;
-	}
-</style>
-
-<?php
-}
-
-function jetpack_verification_tool_box() {
-	global $current_user;
-
-	/**
-	 * Decide whether Site Verification tools be added to the Tools menu.
-	 *
-	 * @module verification-tools
-	 *
-	 * @since 3.0.0
-	 *
-	 * @param bool true Should the Site Verification tools be added to the Tools menu.
-	 */
-	if ( ! apply_filters( 'jetpack_enable_site_verification', true ) )
```

Comment:
We should note in the changelog that this filter is being deprecated.

Reply:
will do!

Reply:
@cbauerman you can do one better and add it to this list of deprecated hooks 

https://github.com/Automattic/jetpack/blob/master/class.jetpack.php#L6597